modules/delegates/components/DelegateDetail.tsx

```diff
@@ -20,9 +25,28 @@ export function DelegateDetail({ delegate, stats }: PropTypes): React.ReactEleme
 
   const { voteDelegateAddress } = delegate;
 
+
+  const tabTitles = [
+    delegate.status === DelegateStatusEnum.recognized ? 'Delegate credentials' : null,
+    'Participation metrics',
+    'Voting History'
+  ].filter(i => !!i) as string[];
```

Comment:
We should be consistent with casing, so capitalize `Delegate Credentials` and `Participation Metrics`